src/pkg/repository/repository.go

```diff
@@ -358,6 +361,20 @@ func (r repository) BackupDetails(
 		return nil, nil, errs.Fail(err)
 	}
 
+	// Retroactively fill in isMeta information for items in older
+	// backup versions without that info
+	// version.Restore2 introduces the IsMeta flag, so only v1 needs a check.
+	if b.Version == version.Restore1DataAndMetaFiles {
```

Comment:
Current version in prod is version 2, the change being it has name in meta. You should be able to see this if you check the version number of the current backups. This is why I had > and < checks.

Reply:
Also, if you see `Restore2IsMetaMarker` is current set to 3. So we need both 1 and 2 to run this.